manifests/aws-efs-csi-driver-operator.package.yaml

```diff
@@ -0,0 +1,4 @@
+packageName: aws-efs-csi-driver-operator
+channels:
+- name: "4.9"
+  currentCSV: aws-efs-csi-driver-operator.v4.9.0
```

Comment:
Lets move away from channel named 4.9 4.10 etc. This is not important anymore, because index images are built from branches and only contain manifest that belong to that branch. This way - users don't have to change channel anymore on OCP upgrades

Reply:
what should be the channel name? "stable"?

Reply:
yep

Reply:
I talked to ART, given the tight schedule,

> let's do it the "traditional way" just to have them ready for QA and in the background we can explore the move to stable

Tracking JIRA: https://issues.redhat.com/browse/ART-3107